libraries/core-react/src/components/Input/Input.tsx

```diff
@@ -3,35 +3,50 @@ import { ElementType, InputHTMLAttributes } from 'react'
 import styled, { css } from 'styled-components'
 import { InputVariantProps, input as tokens } from './Input.tokens'
 import { typographyTemplate, spacingsTemplate } from '@utils'
+import { InputWithAdornments } from '../TextField/InputWrapper'
```

Comment:
> Reviewed in pair by @mimarz & @pomfrida.
> 
> Lots of good changes here! We like the way you wrote token values in css some places 😍
> Mostly suggestions on cleaner code and some new stuff for discussion 😅
> 
> We also found a small visual bug when using multilines with scroll-bar and `box-shadow` as border. Not breaking (and in production). If you know of a fix for it, or else we can live with it I guess.
> 
> ![image](https://user-images.githubusercontent.com/1070981/111137418-0cfa4080-857f-11eb-907a-fdbf9bb87e2f.png)

👍  for pair reviews!

My hope for the future would actually be that we suggest using an icon beside the helper text and not inside the text area 🙈  Or at least limit the use of icons inside text area. I think this is a hard one to fix. I actually see the Input and TextArea as two different components so it might be an idea to split this. 🤔 

In which browser btw? It doesn't look that bad in my Chrome, did you add props or styles to the text area?

Reply:
> Suggest avoiding dependency on `Textfield` in `Input` for separation of concern.
> 
> The same should be achievable by extending `Input` with `styled()` inside the `InputWrapper`.
> 
> Quick draft of extend;
> 
> ```ts
> const StyledInput = styled(Input)`
>   ${InputWithAdornments} & {
>     outline: none;
>   }
>   ${InputWithAdornments} &:active,
>     ${InputWithAdornments} &:focus {
>     outline: none;
>     box-shadow: none;
>   }
> `
> ```

Yes, I agree that it is a bit over engineered as it is now, but that was one of those things I wanted to streamline if we add tokens for Input :) So it's kind of in the middle of something 😂  But I'll remove it for now.